mlrun/db/base.py

```diff
@@ -624,3 +624,11 @@ def set_run_notifications(
 
     def watch_log(self, uid, project="", watch=True, offset=0):
         pass
+
+    @abstractmethod
+    def get_datastore_profile(self, decorated_profile_name: str):
```

Comment:
> Signature of method 'SQLRunDB.get_datastore_profile()' does not match signature of the base method in class 'RunDBInterface' 

Same for all other implementations, so it's probably the abstract signature here that needs to be fixed.